lib/linguist/languages.yml

```diff
@@ -3592,6 +3592,7 @@ Python:
   - ".gclient"
   - BUCK
   - BUILD
+  - BUILD.bazel
   - SConscript
   - SConstruct
   - Snakefile
```

Comment:
Should this be including `WORKSPACE.bazel` files too?

Reply:
No need.